xandikos/access.py

```diff
@@ -22,7 +22,6 @@
 See http://www.webdav.org/specs/rfc3744.html
 """
 
-from defusedxml.ElementTree import fromstring as xmlparse
```

Comment:
I don't quite understand how importing this (but not referencing it) affects the rest of the code, can you explain that a bit? (I'm not arguing, but really want to understand this).